src/main/java/core/basesyntax/Worker.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class Worker {
```

Comment:
for this task it's redundant to create Worker class, you code gets bigger

Reply:
removed. done